tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -526,6 +526,14 @@ describe(__filename, () => {
     expect(root.find('.AddonReviewListItem-begin-reply')).toHaveLength(0);
   });
 
+  it('hides the reply-to-review link on the developer reply', () => {
+    const developerUserId = 3321;
+    const { addon } = signInAsAddonDeveloper({ developerUserId });
+    const { reply } = _setReviewReply();
+    const root = renderReply({ reply, addon });
```

Comment:
Because all we are checking is that the reply link is not displayed for any reply, we don't really need to create an `addon` or a `reply` for this test. I think we can just call `renderReply()` as is done in the test for `it('hides ratings for replies')`.

Reply:
I initially was wondering if we needed a test for this b/c i do see a similar test to what you suggest above but the problem was it wasn't failing there (and we would expect that it would fail - right? - since it was showing this link in the reply). This test always passed. 

With the code changes the only way i could get it to both pass and fail as expected (for testing purposes) was to log in as a developer

I think i can clean it up tho if that helps:

```javascript
  it('hides the reply-to-review link on the developer reply', () => {
      const developerUserId = 3321;
      const { addon } = signInAsAddonDeveloper({ developerUserId });
      const root = renderReply({ addon });
      expect(root.find('.AddonReviewListItem-begin-reply')).toHaveLength(0);
    });
```
i *think* this is b/c renderReply sets up the reply as a regular user so given that we wouldn't ever see this so will always be 0 like you noted above

let me know if that makes any sense or still if there is a better way to address the test : )

Reply:
@bobsilverberg  - hopefully i am not over-pinging you now : ) but please review partial update and comment above regarding your suggestion

Reply:
Ah, I hadn't thought that through. I just saw the code and thought we didn't need it, but yeah, if the reply link won't render without a signed-in developer then we definitely need it. Good job making sure the test fails without your patch, as well as passes with your patch!